src/App.js

```diff
@@ -24,10 +25,12 @@ class App extends Component {
 
   componentDidMount() {
     this.getData('nflTeams').then((data) => {
-      this.setState({
-        nflTeams: data.nflTeams,
-        isLoading: false
-      })
+      setTimeout(() => {
+        this.setState({
+          nflTeams: data.nflTeams,
+          isLoading: false
+        })
+      }, 1000);
```

Comment:
Very cool delay.  Might be harder to test

Reply:
It's going to be hard to test regardless, but the delay could potentially help